### PR TITLE
add access permission on generic assay meta endpoint

### PR DESCRIPTION
**Pull request by dippindots**
add access permission on generic assay meta endpoint

### CODE REVIEW

File: web/src/main/java/org/cbioportal/web/util/InvolvedCancerStudyExtractorInterceptor.java [hunk 1]
```diff
@@ -103,6 +104,7 @@
     public static final String TREATMENT_FETCH_PATH = "/treatments/fetch";
     public static final String STRUCTURAL_VARIANT_FETCH_PATH = "/structuralvariant/fetch";
     public static final String GENERIC_ASSAY_DATA_MULTIPLE_STUDY_FETCH_PATH = "/generic_assay_data/fetch";
+    public static final String GENERIC_ASSAY_META_FETCH_PATH = "/generic_assay_meta/fetch";
```
Comment (inodb) [inline web/src/main/java/org/cbioportal/web/util/InvolvedCancerStudyExtractorInterceptor.java]: @dippindots just noticed these `gneric_assay` paths have underscores in the name while other paths do not. Prolly good to be consistent?

Comment (inodb) [reply web/src/main/java/org/cbioportal/web/util/InvolvedCancerStudyExtractorInterceptor.java]: we can fix it later but just FYI

**Comment by inodb** [comment]
This is a temp fix to address auth issues, will be properly fixed later

**Comment by dippindots** [comment]
@inodb Just want to mention that this is not a temp fix, already updated properly.

**Pull request merged**